src/data/polynomial/degree.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Chris Hughes, Johannes Hölzl, Scott Morrison, Jens Wagemaker
 -/
 import data.polynomial.eval
+import data.polynomial.derivative
```

Comment:
Do you need this import?

Reply:
Sorry, don’t need this. Another line I forgot to delete.

Reply:
I am current out and don’t have access to a computer, I would delete it once I have access to a computer.